src/components/calcite-slider/calcite-slider.tsx

```diff
@@ -747,8 +744,12 @@ export class CalciteSlider implements LabelableComponent {
         prop = closerToMax || position > this.maxValue ? "maxValue" : "minValue";
       }
     }
-    this[prop] = this.clamp(position, prop);
+    this.lastDragPropValue = this[prop];
     this.dragStart(prop);
+    const isThumbActive = this.el.shadowRoot.querySelector(".thumb:active");
```

Comment:
@anveshmekala this still needs to change to not query the shadowRoot every time a render occurs.

Instead, store the active state value in the class.

Reply:
@driskull This query only happens on `pointerdown`, so it doesn't have a noticeable hit on performance.

Reply:
Ok. I think as a best practice we should avoid querying the DOM/shadowRoot when possible though.

Reply:
Agreed, but I think slider needs a bigger refactor before doing so (e.g., consolidating handle rendering, improving/lessening queries on post render).

Reply:
Sounds good. Ignore this feedback @anveshmekala :)